lms/djangoapps/courseware/features/lti.feature

```diff
@@ -48,7 +48,7 @@ Feature: LMS.LTI component
   | False              | 10     | True      | True      |
   And I submit answer to LTI 1 question
   And I click on the "Progress" tab
-  Then I see text "Problem Scores: 5/10"
+  Then I see text "5/10"
```

Comment:
I had removed it trying to figure out the lettuce tests, which were failing to something else. This needs to be replaced. I'll do it now.